source/css/sass/mixins/_types.scss

```diff
@@ -0,0 +1,35 @@
+@import "../variables/font";
+@import "../vendor/sass-rem/rem";
+
+$rem-baseline: $font-size-base;
+$rem-fallback: true;
+
+@mixin safe-rem($properties, $values...) {
```

Comment:
The library produces fallbacks when there's no need (ie there's no pixel value). Will open a PR upstream when I have this working.

Reply:
Opened https://github.com/pierreburel/sass-rem/pull/12.